src/Adapter/Product/AdminProductWrapper.php

```diff
@@ -78,18 +78,24 @@ class AdminProductWrapper
      */
     private $employeeAssociatedShops;
 
+    /**
+     * @var FloatParser
+     */
+    private $floatParser;
+
     /**
      * Constructor : Inject Symfony\Component\Translation Translator.
      *
      * @param object $translator
      * @param array $employeeAssociatedShops
      * @param Locale $locale
      */
-    public function __construct($translator, array $employeeAssociatedShops, Locale $locale)
+    public function __construct($translator, array $employeeAssociatedShops, Locale $locale, FloatParser $floatParser)
```

Comment:
This is a breaking change, I guess you can just add a hard dependency and instantiate in the constructor.

Reply:
Oh I see why you're injecting it now